packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -30,6 +30,11 @@ typedef DismissDirectionCallback = void Function(DismissDirection direction);
 /// Used by [Dismissible.confirmDismiss].
 typedef ConfirmDismissCallback = Future<bool?> Function(DismissDirection direction);
 
+/// Signature used by [Dismissible] to indicate whether the dismiss threshold has
+/// been reached, was previously reached
+/// Used by [Dismissible.onDismissUpdate].
+typedef DismissUpdateCallback = void Function(DismissDirection direction, DismissUpdate dismissUpdate);
```

Comment:
```suggestion
typedef DismissUpdateCallback = void Function(DismissUpdate dismissUpdate);
```
Since we already have a detail class, move the direction to details.